detections/cloud/cisco_isovalent___detect_shell_execution.yml

```
@@ -0,0 +1,57 @@
+name: Cisco Isovalent - Detect Shell Execution
```

You do not need the `detect` keyword as it's implied by the detection itself.

Make the necessary changes in the name/filter

```suggestion
name: Cisco Isovalent - Shell Execution
```